package.json

```diff
@@ -17,7 +17,7 @@
         "@babel/core": "^7.9.6",
         "@babel/register": "^7.9.0",
         "@zeit/ncc": "^0.22.1",
-        "actions-utils": "Khan/actions-utils#v1.2.4",
+        "actions-utils": "Khan/actions-utils#35a4ac8366238b6acd95431db8312f017a69bccc",
```

Comment:
does this want to be 1.2.5 to match https://github.com/Khan/flow-coverage-action/pull/9?